packages/components/pie-button/src/index.ts

```diff
@@ -197,8 +213,9 @@ export class PieButton extends FormControlMixin(LitElement) implements ButtonPro
      */
     private renderSpinner (): TemplateResult {
         const { size, variant, disabled } = this;
-        const spinnerSize = size && size.includes('small') ? 'small' : 'medium'; // includes("small") matches for any small size value and xsmall
-        let spinnerVariant;
+
+        const spinnerSize: SpinnerProps['size'] = size && size.includes('small') ? 'small' : 'medium'; // includes("small") matches for any small size value and xsmall
```

Comment:
Importing the type for child components actually works pretty nicely

Reply:
love it

Reply:
We already have it in `dependencies` so shouldn't add anything that isn't already there 👍🏻